pack/tdc/tdcp.json

```diff
@@ -0,0 +1,553 @@
+[
+    {
+        "back_flavor": "",
+        "back_text": "",
```

Comment:
```suggestion
        "back_text": "<b>This is placeholder back text until the card is revealed.</b>",
```

Reply:
Can you pop this for the empty fields on investigators so that Kam can merge this PR?